doc/content/ttn-lw-cli/ttn-lw-cli.md

```diff
@@ -36,10 +36,10 @@ The Things Industries Command-line Interface
       --output-format string                            Output format (default "json")
       --packet-broker-agent-grpc-address string         Packet Broker Agent address (default "localhost:8884")
       --qr-code-generator-grpc-address string           QR Code Generator address (default "localhost:8884")
-      --retry-config.default-timeout duration           Default timeout between retry attempts (default 100ms)
```

Comment:
Do we have a CLI compatibility issue @adriansmares ?

Reply:
This was intentionally changed in https://github.com/TheThingsNetwork/lorawan-stack/pull/5624 . The original field style was inconsistent (`-config` suffix, and `_` being used instead of `-`) and contained a misspelling, so we decided to fix this since the settings themselves are very unlikely to be in used at all.